lib/services/keyVault/lib/models/certificateBundle.js

```diff
@@ -87,7 +87,7 @@ var models = require('./index');
  * X509 component of a certificate.
  * 
  * @member {string} [policy.issuerReference.name] Name of the referenced
- * issuer object.
+ * issuer object e.g. 'Self', 'Unknown'.
```

Comment:
>issuer object e.g. 'Self', 'Unknown'. [](start = 3, length = 37)

Either name of the referenced issuer object or one of the reserved name of 'Self' or 'Unknown'